packages/hub/src/graphql/builder.ts

```diff
@@ -76,13 +82,21 @@ export const builder = new SchemaBuilder<HubSchemaTypes>({
       // See also: `isRootUser` function in `types/User.ts`.
       return !!(email && process.env.ROOT_EMAILS?.includes(email));
     },
+    isGroupAdmin: async (groupId) => {
+      const myMembership = await getMyMembershipById(groupId, context.session);
+      return myMembership?.role === "Admin";
+    },
+    isGroupAdminBySlug: async (groupSlug) => {
+      const myMembership = await getMyMembershipBySlug(
+        groupSlug,
+        context.session
+      );
+      return myMembership?.role === "Admin";
+    },
```

Comment:
Having more authScopes like these will allow us to simplify many other graphql fields (especially mutations) definitions in the future. I haven't done it here but it's now cleaner to me how to do that.